src/accelerate/accelerator.py

```diff
@@ -71,6 +73,21 @@
         DummyScheduler,
     )
 
+if is_megatron_lm_available():
+    from .utils import (
+        MegatronEngine,
+        MegatronLMDummyDataLoader,
+        MegatronLMDummyScheduler,
+        MegatronLMOptimizerWrapper,
+        MegatronLMSchedulerWrapper,
+    )
+    from .utils import initialize as megatron_lm_initialize
+    from .utils import prepare_data_loader as megatron_lm_prepare_data_loader
+    from .utils import prepare_model as megatron_lm_prepare_model
+    from .utils import prepare_optimizer as megatron_lm_prepare_optimizer
+    from .utils import prepare_scheduler as megatron_lm_prepare_scheduler
```

Comment:
The aliases should be done in the `.utils` module: if we have another integration like this, we won't be able to have both names.

Reply:
Done.